.gitignore

```diff
@@ -1,3 +1,4 @@
+*.swp
```

Comment:
why did this end up here?

Reply:
Because I edit these files using vim. 
I know that I should have this in global gitingore, but I think we should have default configuration fits for most editors. 

I could revert this.

Reply:
fine by me to leave it in, but I think I already have this in my global ignore :P